network/openvswitch_port.py

```diff
@@ -47,44 +54,188 @@
         default: 5
         description:
             - How long to wait for ovs-vswitchd to respond
+    iface-id:
+        required: false
```

Comment:
Brian,
What is the process here? How should I address these changes?
-Mark 
From: Brian Coca 
Sent: Friday, July 10, 2015 3:36 PM
To: ansible/ansible-modules-extras 
Cc: Mark Hamilton 
Subject: Re: [ansible-modules-extras] Added support to assign attached mac address interface id and port options. (#702)

In network/openvswitch_port.py:

> @@ -47,44 +54,188 @@
>          default: 5
>          description:
>              - How long to wait for ovs-vswitchd to respond
> +    iface-id:
> +        required: false
when required: false, default must be specified

—
Reply to this email directly or view it on GitHub.